packages/exceptions/src/http/UriException.ts

```diff
@@ -1,6 +1,7 @@
-import { RuntimeException } from '@monument/core';
+import { RuntimeException } from '../base/RuntimeException';
 
 /**
+ * @todo need to consider removing it
```

Comment:
`UriException` is base exception for family of URI exceptions, it should stay

Reply:
Removed todos. As we discussed in a call, I'd research the best way to organize scope of specialized exceptions for URLs and HTTP:ok_hand: